templates/parachain/runtime/src/genesis_config_presets.rs

```diff
@@ -76,6 +76,8 @@ fn local_testnet_genesis() -> Value {
 		],
 		Sr25519Keyring::well_known().map(|k| k.to_account_id()).collect(),
 		Sr25519Keyring::Alice.to_account_id(),
+    // TODO: this is super opaque, how should one know they should configure this? add to
```

Comment:
Should we mention it in an issue to avoid forgetting about it ?

Reply:
Yeah, @iulianbarbu I believe this is something that should be covered in the parachain template docs/README, can you note it please?